Transformers/ServiceNowFormDataTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace EMS\SubmissionBundle\Transformers;
+
+class ServiceNowFormDataTransformer extends AbstractFormDataTransformer
+{
+    public function transform(): array
+    {
+        $data = $this->form->getData();
+
+        return $this->mapNewFieldTimeFormat($data);
+    }
+
+    private function mapNewFieldTimeFormat(array $data): array
+    {
+        $newFields = $this->getNewFieldTimeFormatValues();
+
+        foreach ($newFields as $k => $v) {
+            $data = $this->arrayReplace($data, $k, $v);
+        }
+
+        return $data;
+    }
+
+    private function getNewFieldTimeFormatValues(): array
+    {
+        $newFields = [];
+
+        $types = [
```

Comment:
We hard code only one type, there is no need for an array of types at this point.